packages/contracts/src/core/dao/IDAO.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity 0.8.17;
 
-/// @title IDAO
+/// @title IDAO v1.1.0
```

Comment:
Do we need to create a new version of the interface, if nothing changed?

Reply:
Oh, ok then 👍 
Let's make sure that this change is on the changelog, feel free to resolve